api/python/Makefile

```
@@ -4,3 +4,6 @@ test:
 
 install-local:
 	pip install -e .[tests]
+
+testdocs:
+	cd ../../testdocs && poetry install && poetry run pytest --codeblocks ../docs
```

Why not have a newline here?
```suggestion
	cd ../../testdocs && poetry install && poetry run pytest --codeblocks ../docs

```